course05/homework01/freekpaans/string.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <string>
+#include <ostream>
+#include <istream>
+
+class String
+{
+public:
+	String();
+	String(std::string);
```

Comment:
const ref, otherwise you copy and this might allocate/cause some unnecessary work